openmmapi/include/openmm/TabulatedFunction.h

```diff
@@ -90,6 +91,12 @@ class OPENMM_EXPORT Continuous1DFunction : public TabulatedFunction {
      * @param[out] max            the value of x corresponding to the last element of values
      */
     void getFunctionParameters(std::vector<double>& values, double& min, double& max) const;
+    /**
+     * Get the periodicity status of the tabulated function.
+     *
+     * @param periodic       whether the function is periodic
+     */
+    void getPeriodic(bool& periodic) const;
```

Comment:
It can just return the value directly: `bool getPeriodic()`.  Returning values through references is only needed when a function has to return multiple values.

Reply:
Done!